livedata/build.gradle

```diff
@@ -139,7 +139,7 @@ afterEvaluate {
                 // Adds sources as separate jars.
                 artifact androidSourcesJar
                 // Custom attributes of the publication
-                groupId = project.group
+                groupId = 'com.github.getstream'
```

Comment:
Why are we changing it?

Reply:
If `publishToMavenLocal` is done with 'com.github.GetStream' group there is a metadata error while fetching the artifact in the target project. Or is it only locally on my end?

Reply:
I didn't have this problem before.
BTW, Can't you change the `project.group` value?

Reply:
Not sure how jitpack will react. I will probably get stuck finding repo on github